### ISSUE TITLE
[UWP] Could not load file or assembly System.IO.FileSystem

### ISSUE THREAD

**Issue by asandoval4wall** [dotnet/core#945]
# [UWP] Could not load file or assembly 'System.IO.FileSystem

# General

I'm creating a UWPjs app with a C# UWP runtime and it fails to create files because it fails to find System.IO.FileSystem. I get no errors at build time, only at runtime. Any info helps, thanks.

- Target Version - Windows 10 Creators Update (10.0, Build 15063)
- OS - Win 10 Pro, Version 10.0.15063 Build 15063

# Code
![image](https://user-images.githubusercontent.com/29235978/30076142-dd188ee2-922c-11e7-9937-e47bffe5ffe2.png)

# Exception
Unhandled exception at line 35, column 17 in ms-appx://fbca3db9-8274-4457-97e8-a4b241b376c1/dist/atom.js
0x80070002 - JavaScript runtime error: The system cannot find the file specified.

System.IO.FileNotFoundException: Could not load file or assembly 'System.IO.FileSystem, Version=4.0.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.
   at uwpAppJs.Example1.CreateFile()
WinRT information: System.IO.FileNotFoundException: Could not load file or assembly 'System.IO.FileSystem, Version=4.0.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.
   at uwpAppJs.Example1.CreateFile() occurred

**Comment by asandoval4wall** [comment]
I've come to the conclusion that I've misunderstood how I should have approached this problem. I assume the exception I get is by design. You get limited ability to do IO operations for a reason. (may not be a good enough reason for me but it's a reason nonetheless).

See https://docs.microsoft.com/en-us/windows/uwp/files/file-access-permissions for doing limited IO operations with UWPjs projects, be it someone else having this issue.

In short you can actually do IO based things via winjs which is actually nicer than what I was doing.
